components/src/dynamo/common/tests/multimodal/test_image_loader.py

```
@@ -179,46 +147,35 @@ async def test_waiter_cancellation_does_not_cancel_shared_task(
-    image_path = tmp_path / "secret.png"
-    Image.new("RGB", (1, 1), color="red").save(image_path, format="PNG")
-
-    with pytest.raises(ValueError, match="Invalid image source scheme"):
-        await loader.load_image(url_factory(image_path))
+    Validator behavior is covered in ``test_url_validator.py``;
+    per-hop SSRF revalidation in ``http/test_http_backends.py``.
+    """
+    strict_loader = ImageLoader(
+        cache_size=4, http_timeout=30.0, url_policy=UrlValidationPolicy()
+    )
+    with pytest.raises(ValueError, match="scheme|not allowed"):
```

Ruff (0.15.12)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)